src/definitions/modules/flyout.js

```diff
@@ -1303,10 +1310,11 @@
             }
         });
 
-        return (returnedValue !== undefined)
+        return returnedValue !== undefined
             ? returnedValue
             : this;
     };
+    $.flyout = $.fn.flyout;
```

Comment:
Ok, i see this is instead added here... what's the benefit?

Reply:
yes this change is intended, see https://github.com/fomantic/Fomantic-UI/pull/2589/commits/e4333cfff8c6fd1cf856a0b5b83658c3327785d9, it is a recommended practice to never assign more than one variable in one line to improve readability